main/src/ca/uwaterloo/flix/api/lsp/provider/completion/Completion.scala

```diff
@@ -46,6 +46,24 @@ sealed trait Completion {
         sortText = Priority.toSortText(priority, name),
         textEdit = TextEdit(context.range, s"$name "),
         kind = CompletionItemKind.Keyword)
+    case Completion.KeywordLiteralCompletion(name, priority) =>
```

Comment:
Lets add a new linebreak before each case to give some air.

Reply:
Yeah, I agree. I have considered that as well. It's kinda overwhelming right now